### ISSUE TITLE
bug: youtube music not playing

### ISSUE THREAD

**Issue by Sub123945** [ReVanced/revanced-patches#4103]
### Bug description

https://github.com/user-attachments/assets/865cef31-a33d-435e-b370-35cbc96f0156
YouTube music not playing 

### Error logs

```shell

```

### Acknowledgements

- [x] I have checked all open and closed bug reports and this is not a duplicate.
- [x] I have chosen an appropriate title.
- [x] All requested information has been provided properly.

**Comment by Sub123945** [comment]
![Image](https://github.com/user-attachments/assets/5d282fb7-d0ae-49da-bf28-6fab868afb2e)

**Comment by Karthik-Subbu** [comment]
Few observations: 

YT music 7.26.51, patch version 5.3.0 spoof video streams excluded: Playback stutters.  Loading of action bar, navigation components is taking time. Playback starts after nearly half a minute.

YT music 7.26.51, patch version 5.3.0 spoof video streams included: Playback is not initiated.

YT music 7.26.51, patch version 5.2.4 dev.3 (before introduction of spoof video streams patch): No playback issues. Instant playback.

I think if at all a playback issue exists in YT music for selected users at the moment, spoof client patch with IOS client will suffice at the moment. (Please see https://github.com/inotia00/ReVanced_Extended/issues/2484 )

Further in new YT music version 7.29.52, I am seeing a new style navigation bar with 10 second backward and 30 second forward options. May be A/B testing.

**Comment by oSumAtrIX** [comment]
[Follow these steps and report back](https://i.imgur.com/cmE5NSs.png)

**Comment by Sub123945** [comment]
@LisoUseInAIKyrios I don't have a computer.

**Comment by Sub123945** [comment]
Can you send me guidelines?

**Comment by 3092106782** [comment]
***@***.***

***@***.***

**Comment by Sub123945** [comment]
Sorry I'm tried but failed. Please take help from somebody.

**Comment by Karthik-Subbu** [comment]
Try this file.

https://we.tl/t-4mmk1PwByv

For testing purpose only. Always patch by yourself.

**Comment by Sub123945** [comment]
A song play just now its takes approximately half minutes to play

**Comment by Karthik-Subbu** [comment]
Yeah the same situation I was facing this evening. The video was loading after about half a minute. Then later on it was fine. I had to delete the comment as the issue was not repeatable. Sharing files may be against guidelines. Can you do this  - patch 7.16.53 using RVX manager, enable spoof client and see the playback behaviour.

**Comment by Sub123945** [comment]
@Karthik-Subbu yes I'm patched, there is no issue, music are playing properly.![Image](https://github.com/user-attachments/assets/a8239641-b555-4b44-a4b7-ba8cc64073ad)
![Image](https://github.com/user-attachments/assets/e84a8aed-c82e-4177-9717-f145868bcea2)

**Comment by oSumAtrIX** [comment]
Unofficial patches are unrelated here.

**Comment by k3lt** [comment]
Same issue here, unpatched Youtube Music players normal (with ads..)

**Comment by Loup-Garou911XD** [comment]
@Sub123945 are you using cloudflare dns?
I was having the same issue and disabling the dns fixed it for me.

**Comment by Loup-Garou911XD** [comment]
Also it works fine on my wifi

**Comment by Sub123945** [comment]
@Loup-Garou911XD I'm using adguard dns

**Comment by Sub123945** [comment]
> > [@Loup-Garou911XD](https://github.com/Loup-Garou911XD) I'm using adguard dns
> 
> Have you tested without connect to any DNS Services?

I'm just tested without connect adguard dns  but still loading problem

**Comment by Sub123945** [comment]
> > > [@Loup-Garou911XD](https://github.com/Loup-Garou911XD) I'm using adguard dns
> > 
> > 
> > Have you tested without connect to any DNS Services?
> 
> I'm just tested without connect adguard dns but still loading problem

When I turns off & on mobile data then song playshttps://github.com/user-attachments/assets/7e1d28f5-b8b3-4781-b234-32a508e2152e
Watch the video

**Comment by Loup-Garou911XD** [comment]
> > > [@Loup-Garou911XD](https://github.com/Loup-Garou911XD) I'm using adguard dns
> > 
> > Have you tested without connect to any DNS Services?
> 
> I'm just tested without connect adguard dns  but still loading problem 

Okay maybe then my case was a false positive and something else fixed the issue for me 
But now I am having the issue again and dns doesn't fix it

**Comment by Sub123945** [comment]
> > > > [@Loup-Garou911XD](https://github.com/Loup-Garou911XD) I'm using adguard dns
> > > 
> > > 
> > > Have you tested without connect to any DNS Services?
> > 
> > 
> > I'm just tested without connect adguard dns  but still loading problem
> 
> Okay maybe then my case was a false positive and something else fixed the issue for me But now I am having the issue again and dns doesn't fix it But i


> > > > [@Loup-Garou911XD](https://github.com/Loup-Garou911XD) I'm using adguard dns
> > > 
> > > 
> > > Have you tested without connect to any DNS Services?
> > 
> > 
> > I'm just tested without connect adguard dns but still loading problem
> 
> When I turns off & on mobile data then song plays
> 
> Watch the video https://github.com/user-attachments/assets/883a6b85-92f7-4e25-b9ad-e030b77244ef

@oSumAtrIX 
@cyberboh 
@LisoUseInAIKyrios 
@Karthik-Subbu 
Watch the video

**Comment by Sub123945** [comment]
I'm only use mobile data.
Force stopped the app then again opend and played.
Still the issue.

**Comment by Karthik-Subbu** [comment]
> @Sub123945 If you have a computer, then try patching using command prompt and CLI tools using this release: [patches-5.4.0-yt-music-spoof-ios-music.rvp.zip](https://github.com/user-attachments/files/18094728/patches-5.4.0-yt-music-spoof-ios-music.rvp.zip)
> 
> It's the changes as found [here](https://github.com/LisoUseInAIKyrios/revanced-patches/commit/3774db0b1d87873d4b8ca0ef197b20683ff75e20)
> 
> And anyone else can try patching with this change.
> 
> 
> 

This spoof video streams patch uses only IOS music client for YT music?  Is the language parameter needed?
